preferences/about.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+// Gnome imports
+const { Adw, GObject, Gtk } = imports.gi;
+
+// Extension imports
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+
+const _ = imports.gettext.domain(Me.metadata.uuid).gettext;
+
+// Application imports
+const Msgs = Me.imports.messages;
+
+var AboutPage = GObject.registerClass(class AboutPage extends Adw.PreferencesPage {
+  _init({ settings }) {
+    super._init({ title: _("About"), icon_name: 'forge-logo-symbolic' });
+    // TODO: Adw.AboutWindow
```

Comment:
if developer and experimental pages were combined into a single "settings" page (or something like that), then the AdwAboutWindow could be activated by a suffix button in the header for that page

Reply:
For the Experimental, I am thinking these values need to go into proper sections, and have a superscript icon to denote they are experimental, so it much cleaner to remove the Experimental icon later. I was planning it on #217

Reply:
I made some minor changes here, but we can iterate on the designs in #217 in a later PR